### ISSUE TITLE
Newsletter page

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/AAPB2#1222]
Done when:
- [x] Newsletter signup page is created using MailChimp code provided below
- [x] Link to page (as "Newsletter Signup") is added to the About Us header menu (above Contact Us)
- [x] Link to page (as "Newsletter Signup") is added to the footer (under Donate)
- [x] Link to page is added to the Contact Us Page (saying Subscribe to the AAPB mailing list)
- [x] Link to page is added to Donate Page (saying Subscribe to the AAPB mailing list)

Here the the code to put on a new page on the website for the Newsletter sign-up:
```
<!-- Begin MailChimp Signup Form -->
<link href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css" rel="stylesheet" type="text/css">
<style type="text/css">
	#mc_embed_signup{background:#fff; clear:left; font:14px Helvetica,Arial,sans-serif; }
	/* Add your own MailChimp form style overrides in your site stylesheet or in this style block.
	   We recommend moving this block and the preceding CSS link to the HEAD of your HTML file. */
</style>
<div id="mc_embed_signup">
<form action="//americanarchive.us15.list-manage.com/subscribe/post?u=c3e1a540057b82c5b78a914f1&amp;id=cd3c0031cf" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
    <div id="mc_embed_signup_scroll">
	<h2>Subscribe to the American Archive of Public Broadcasting mailing list</h2>
<div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
<div class="mc-field-group">
	<label for="mce-EMAIL">Email Address  <span class="asterisk">*</span>
</label>
	<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
</div>
<div class="mc-field-group">
	<label for="mce-FNAME">First Name </label>
	<input type="text" value="" name="FNAME" class="" id="mce-FNAME">
</div>
<div class="mc-field-group">
	<label for="mce-LNAME">Last Name </label>
	<input type="text" value="" name="LNAME" class="" id="mce-LNAME">
</div>
<div class="mc-field-group">
	<label for="mce-MMERGE3">How did you hear about the AAPB? </label>
	<input type="text" value="" name="MMERGE3" class="" id="mce-MMERGE3">
</div>
<div class="mc-field-group input-group">
    <strong>Email Format </strong>
    <ul><li><input type="radio" value="html" name="EMAILTYPE" id="mce-EMAILTYPE-0"><label for="mce-EMAILTYPE-0">html</label></li>
<li><input type="radio" value="text" name="EMAILTYPE" id="mce-EMAILTYPE-1"><label for="mce-EMAILTYPE-1">text</label></li>
</ul>
</div>
	<div id="mce-responses" class="clear">
		<div class="response" id="mce-error-response" style="display:none"></div>
		<div class="response" id="mce-success-response" style="display:none"></div>
	</div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_c3e1a540057b82c5b78a914f1_cd3c0031cf" tabindex="-1" value=""></div>
    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
    </div>
</form>
</div>
<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='MMERGE3';ftypes[3]='text';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
<!--End mc_embed_signup-->
```

**Comment by jasoncorum** [comment]
@sroosa - how does this look? think we need any custom styling?

![screen shot 2017-07-27 at 2 41 39 pm](https://user-images.githubusercontent.com/3814190/28686628-d3f3b1a6-72d9-11e7-829a-dcc187b95cdb.png)

**Comment by caseyedavis12** [comment]
@jasoncorum: this looks good to me, but can you move the Newsletter label in the side bar down to below FAQ?

**Comment by jasoncorum** [comment]
@caseyedavis12 - totally! i just set up the page and hadn't added it anywhere in the navigation yet. here it is with "Newsletter" added to the sidebar.

![screen shot 2017-07-27 at 2 48 02 pm](https://user-images.githubusercontent.com/3814190/28686865-b0add9be-72da-11e7-874a-10af39c5f87f.png)

**Comment by caseyedavis12** [comment]
@jasoncorum: Great! This will need a separate ticket, but I noticed something in the screenshots you sent. When you click on an page label in the About menu, the page label is duplicated and put at the top of the menu, and the duplicate label is highlighted. Why is that? I think it should just highlight the original page label and not duplicate it and put it at the top.

**Comment by jasoncorum** [comment]
@caseyedavis12 - that may have come from when we did the page redesign and we just didn't notice until now. I don't think we used to have the title in the thicker purple bar. The "highlighted" text isn't clickable or anything, so it may have just been serving the purpose of being like a page title or something. but, yeah, we can figure out something and ticket it.

in the meantime, is this cool for the location of the subscribe text on the Donate and Contact Us pages?

![screen shot 2017-07-28 at 9 48 55 am](https://user-images.githubusercontent.com/3814190/28720689-e6109316-737b-11e7-84dd-796a8945d4ea.png)

![screen shot 2017-07-28 at 9 49 27 am](https://user-images.githubusercontent.com/3814190/28720705-f720a8f8-737b-11e7-90f9-3d19b5c10de2.png)
